src/main/java/thePackmaster/actions/boardgamepack/DelayedDiceLevelUpAction.java

```diff
@@ -0,0 +1,23 @@
+package thePackmaster.actions.boardgamepack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import thePackmaster.powers.boardgamepack.DicePower;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static thePackmaster.util.Wiz.adp;
+
+public class DelayedDiceLevelUpAction extends AbstractGameAction {
+
+    public DelayedDiceLevelUpAction() {
```

Comment:
Could be an anonymous action, they help me keep the project to a manageable size

Reply:
I'll give that a try